From 3af13842221e3264035c157fb3282f5de43b5c94 Mon Sep 17 00:00:00 2001
From: Sean Hefty <sean.hefty@intel.com>
Date: Mon, 25 Mar 2013 14:04:57 -0700
Subject: [PATCH] Increase size of available file name

ibacm limits the path to the log and lock file to 32 characters.
This is too small for some users.

Signed-off-by: Sean Hefty <sean.hefty@intel.com>
---
 src/acm.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/acm.c b/src/acm.c
index 725cde2..e956b09 100644
--- a/src/acm.c
+++ b/src/acm.c
@@ -3068,7 +3068,7 @@ static void acm_set_options(void)
 {
 	FILE *f;
 	char s[120];
-	char opt[32], value[32];
+	char opt[32], value[256];
 
 	if (!(f = fopen(opts_file, "r")))
 		return;
@@ -3077,7 +3077,7 @@ static void acm_set_options(void)
 		if (s[0] == '#')
 			continue;
 
-		if (sscanf(s, "%32s%32s", opt, value) != 2)
+		if (sscanf(s, "%32s%256s", opt, value) != 2)
 			continue;
 
 		if (!stricmp("log_file", opt))
-- 
1.8.1.2

